src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3962,12 +3962,16 @@
 
     /**
      * Beginning of block comment: '/*'.
-     *
+     * <p>For example:</p>
      * <pre>
-     * +--BLOCK_COMMENT_BEGIN
-     *         |
-     *         +--COMMENT_CONTENT
-     *         +--BLOCK_COMMENT_END
+     * /&#42; Comment content
+     * &#42;/
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * --BLOCK_COMMENT_BEGIN -&gt; /&#42;
+     *    |--COMMENT_CONTENT -&gt;   Comment content\r\n
```

Comment:
There should be only two spaces in `-&gt;   Comment`.

Reply:
Okay I will fix it👍